operations/services.md

```diff
@@ -136,6 +136,8 @@ Negative or zero end offsets of the daily time unit define the daily service as
 
 ## Get all availability blocks
 
+> This feature is being actively developed, features and behavior of this operation might change on short notice.
```

Comment:
@MikeAdamsMews can you pls check if this makes sense to you? Or would you rephrase it?

Reply:
on behalf of @MikeAdamsMews
```suggestion
> This feature is being actively developed, features and behavior of this operation might change at short notice.
```